lib/score/view/change_turn_button.dart

```diff
@@ -1,20 +1,43 @@
 import 'package:flutter/material.dart';
+import 'package:smack_talking_scoreboard_v3/app/bloc/app_state.dart';
+import 'package:smack_talking_scoreboard_v3/score/bloc/scoreboard_events.dart';
+import 'package:smack_talking_scoreboard_v3/score/bloc/scoreboard_state.dart';
+import 'package:smack_talking_scoreboard_v3/score/score.dart';
 import 'package:smack_talking_scoreboard_v3/score/view/ui_components/circular_button.dart';
 
-class ChangeTurnButton extends StatelessWidget {
+class ChangeTurnButton extends StatefulWidget {
   const ChangeTurnButton({
     super.key,
-    required this.onTap,
   });
 
-  final VoidCallback? onTap;
+  @override
+  State<ChangeTurnButton> createState() => _ChangeTurnButtonState();
+}
+
+class _ChangeTurnButtonState extends State<ChangeTurnButton> {
+  @override
+  void initState() {
+    super.initState();
+    _checkIfGameHasWinner();
```

Comment:
this is for edge-case that user has completed a game with a winner, closed the app, and restarted it. Without it, the user would be left seeing the app with the score page in a dead state with no indication as to why